aws/resource_aws_servicecatalog_product_test.go

```diff
@@ -0,0 +1,340 @@
+package aws
+
+import (
+	"fmt"
+	"strings"
+	"testing"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/servicecatalog"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/acctest"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/terraform"
+)
+
+func TestAccAWSServiceCatalogProduct_basic(t *testing.T) {
+	salt := acctest.RandString(5)
```

Comment:
align with other provider tests and use `rName := acctest.RandomWithPrefix("tf-acc-test)` and pass it as resource names instead of interpolating them

Reply:
done